src/api/spec/models/review_spec.rb

```diff
@@ -41,16 +41,17 @@
 
   describe '#accepted_at' do
     let!(:user) { create(:user) }
+    let(:review_state) { :accepted }
```

Comment:
Because it saves me having to use a `let!(:review) { ... }` in each of the four contexts. Since the first three contexts need `state = :accepted` but only the last context needs `state = :new` it saves copying these lines of code. But I dont mind duplicated the code if it makes it easier to understand?